exaudfclient/base/javacontainer/test/cpp/javacontainer_test.cc

```diff
@@ -156,6 +199,47 @@ TEST(JavaContainer, simple_import_script) {
     EXPECT_EQ(vm.getJavaVMInternalStatus().m_jvmOptions, expectedJVMOptions);
 }
 
+TEST(JavaContainer, simple_import_script_with_white_space) {
```

Comment:
does it make sense also to add a test with a quoted import script, because it is legal